lib/nerves_hub_web/channels/device_channel.ex

```diff
@@ -545,4 +560,9 @@ defmodule NervesHubWeb.DeviceChannel do
     |> assign(:device, device)
     |> assign(:deployment_channel, deployment_channel)
   end
+
+  defp device_deployment_change_jitter_seconds() do
+    jitter = Application.get_env(:nerves_hub, :device_deployment_change_jitter_seconds)
+    :rand.uniform(jitter) * 1000
```

Comment:
```suggestion
    :rand.uniform(jitter * 1000)
```
Adding even more jitter since we can. No idea if this helps...

Reply:
Yeah, I'm not sure either. I think we want second separation to let queries complete so we don't overload?

Reply:
I thought the first way selected 1 out of 10 slots to put queries by default. Won't that result in a pattern where queries get issued on 1 second boundaries rather than all over the place? My expectation in other context is that smearing things out is better. What I didn't know is if the database did some smart batching and databases just liked it when you thinned things out by 1/10, for example.

A meta-comment here is that this all seems hard to tune. Wouldn't it be better to queue these up to something that can feed the database at whatever speed it can take?

Reply:
Probably but it would not be as easy to switch to as jittering is, so that's how I started here. We can't change certain deployments at the moment so I was going for something that could work fast, but we're already a week out from that so :shrug: I can figure out how to migrate to a queue that we zip through probably per node since I think this thing will need to communicate with connected device channels

Reply:
Cool. I'm done with my comments. Thanks for the explanations.